sources/Algo/include/ContingencyValidationAlgorithm.h

```diff
@@ -113,6 +113,7 @@ class ContingencyValidationAlgorithmOnNodes {
 
  private:
   ValidContingencies& validContingencies_;  ///< the contingencies being validated by the algorithm
+  const bool notInNetwork_ = false;         ///< boolean determining the component is not modelled as network
```

Comment:
No need to put it as attribute, a local variable is fine

Reply:
ok!